nette/notejam/app/forms/Note/DeleteNoteFormFactory.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Forms\Note;
+
+use App\Model\NoteManager;
+use Nette\Application\UI\Form;
+use Nette\Utils\ArrayHash;
+
+
+class DeleteNoteFormFactory
+{
+
+	/** @var NoteManager */
+	private $noteManager;
+
+	/** @var int */
+	private $id;
+
+	/**
+	 * EditPadFormFactory constructor.
+	 * @param NoteManager $noteManager
+	 */
+	public function __construct(NoteManager $noteManager)
+	{
+		$this->noteManager = $noteManager;
+	}
+
+	/**
+	 * @param int $id
+	 * @return Form
+	 */
+	public function create($id)
+	{
+		$this->id = $id;
```

Comment:
I don't like that the Factory changes it's inner state everytime a `create()` is called. Would this work if you created multiple forms with different ids in one request? Wouldn't the last id override the previous ones?